implementations/rust/ockam/ockam_api/src/nodes/service/relay.rs

```diff
@@ -37,20 +37,27 @@ impl NodeManagerWorker {
     ) -> Result<Response<RelayInfo>, Response<Error>> {
         let CreateRelay {
             address,
-            alias,
+            name,
             authorized,
             relay_address,
         } = create_relay;
+
         match self
             .node_manager
-            .create_relay(ctx, &address, alias, authorized, relay_address)
+            .create_relay(ctx, &address, name.clone(), authorized, relay_address)
             .await
         {
             Ok(body) => Ok(Response::ok().with_headers(req).body(body)),
-            Err(err) => Err(Response::internal_error(
-                req,
-                &format!("Failed to create relay: {}", err),
-            )),
+            Err(err) => match err.code().kind {
+                Kind::Timeout => Err(Response::internal_error(
+                    req,
+                    &format!("Failed to create relay at {address} with name {name}. Make sure you are allowed to create this relay. {err}")
```

Comment:
> How can a user check that they can, or not, create a relay?

I think this would require refactoring the inners of how secure channel are created. I think at some point we know/log whether an identity is authorized to create the secure channel, but we always get a timeout as the final error.

But yeah, I agree that this change may be assuming too much about the underlying error. I think I should rollback it.